modules/ecs/cluster/variables.tf

```diff
@@ -32,4 +32,28 @@ variable "tags" {
   default = {
     terraform = true
   }
+}
+
+variable "app_name" {
```

Comment:
I think this can be a local var we construct in the module.

Reply:
shouldn't we let the thing using the module dictate the name? I'll remove the default cause it shouldn't be there anyways